tests/Broadcasting/AblyBroadcasterTest.php

```diff
@@ -19,14 +20,24 @@ class AblyBroadcasterTest extends TestCase
 
     protected function setUp(): void
     {
+        if (phpversion() >= '8.2') {
```

Comment:
this should use `version_compare` or `\PHP_VERSION_ID`. Comparing version strings alphabetically is wrong for some cases (but I hope you'll be able to re-enable those tests before PHP reaches version `8.10` or `10.0`)

Reply:
Yeah we don't plan on keeping this long